### PR TITLE
The good clown brings +0.1 layer to poddoors

### PR DESCRIPTION
**Pull request by RyanSmake**
Poddoors now finally start above podlocks, so mappers now don't have to adjust this manually every damn time.

### CODE REVIEW

File: code/game/machinery/doors/poddoor.dm [hunk 1]
```diff
@@ -101,6 +101,7 @@
 
 /obj/machinery/door/poddoor/multi_tile // Whoever wrote the old code for multi-tile spesspod doors needs to burn in hell.
 	name = "large pod door"
+	layer = 3.1
```
Comment (Citinited) [inline code/game/machinery/doors/poddoor.dm]: You should use a layer define for this

**Comment by Kyep** [comment]
I know this isn't strictly on topic, but.... if you're going to fix pod doors, could you also fix the bug that causes them to not be opaque when closed? IE when closed you can see through them even if they are not there?

I actually started coding a fix for this myself, but I never got around to finishing it.
You're welcome to use my code as a starting point: https://github.com/ParadiseSS13/Paradise/compare/master...Kyep:poddoor_fix

File: code/game/machinery/doors/poddoor.dm (hunk 1 above)
Comment (RyanSmake) [reply code/game/machinery/doors/poddoor.dm]: Aye aye.

**Comment by RyanSmake** [comment]
I will take a look.

**Comment by tigercat2000** [comment]
Seems right to me

**Pull request merged**